modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/source-builder/xmlSchemaUtil.js

```diff
@@ -95,11 +95,31 @@ function getLocationValue(field, context) {
 										itemChild.childNodes
 									);
 
+									const itemChildNodesAttributes = getChildAttributes(
+										item.childNodes
+									);
+
 									let itemContent;
 
 									if (childNodesAttributes.length) {
 										itemContent = childNodesAttributes;
 									}
+									else if (
+										itemChildNodesAttributes.length
+									) {
+										itemContent = itemChildNodesAttributes;
+
+										if (itemChildNodesAttributes.length) {
+											if (!childContent[item.tagName]) {
+												childContent[item.tagName] = [];
+											}
+
+											childContent[
+												item.tagName
+											] = itemContent;
+										}
+										break;
```

Comment:
@alinedoleron Maybe it would be better to use `return` instead of `break`. What do you think?

Reply:
Hi @Nara, I've changed to return but when I tested I noticed the return not only stops the for, but also leaves the if condition, which is not the behavior we want. So, I'm going back the code to use the break ok? But if you have another suggestion, please let me know.